src/system_simulation/server/simulation_runner_server.rb

```diff
@@ -11,21 +11,21 @@
 post '/update_model' do
   sim = SimulationRunner.new_from_json_export(request.body.read)
   p sim
-  return ""
+  return ''
 end
 
 patch '/update_spring_constants' do
   sim.update_spring_constants(JSON.parse(request.body.read))
-  return ""
+  return ''
 end
 
 patch '/update_mounted_users' do
   sim.update_mounted_users(JSON.parse(request.body.read))
-  return ""
+  return ''
 end
 
-get '/get_period/:node_id' do
-  return_message = { period: sim.get_period(params['node_id']) }
+get '/get_user_stats/:node_id' do
+  return_message = sim.get_user_stats(params['node_id'])
   return_message.to_json
```

Comment:
Will postpone that since we've got an [tello card](https://trello.com/c/wjhuEfE6/548-refine-error-reporting-communicate-exceptions-from-omc-back-to-sketchup) to improve the whole error handling for the communication between `SimulationRunnerClient` and `Server`